assets/js/data/cart/actions.js

```diff
@@ -98,8 +98,6 @@ export function* applyCoupon( couponCode ) {
 	} catch ( error ) {
 		// Store the error message in state.
 		yield receiveError( error );
-		// Finished handling the coupon.
-		yield receiveApplyingCoupon( '' );
```

Comment:
Why was this removed?

Reply:
I'm assuming you saw the comment I [left here](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/1972#discussion_r392603600) about this (since you acknowledged it with a thumbs up emojii). Is that sufficient explanation?